### DISCUSSION TITLE
peacock can not be started after upgrading moose

### DISCUSSION THREAD

**Discussion by zhx999** [idaholab/moose#27317] [Q&A General] [answered]
### Bug Description

Hi there,

I have installed the latest version of moose and pulled the code from github. However, when I try to start peacock in terminal, it shows the following error:

Error starting peacock: Could not import matplotlib

You may need to either create or load an environment providing PyQt, VTK, etc.
The MOOSE development team provides a conda package with the required dependencies:

	mamba create -n peacock moose-peacock python=3.10
	mamba activate peacock

Then run peacock again

### Steps to Reproduce

Run command "peacock" in terminal

### Impact

The postprocessor doesn't work

**Comment by milljm** [comment]
I am afraid this is not going to be something we can change in our code that will fix this issue. Peacock not launching is always going to be some library missing on the failing machine, which needs to be installed. Be it by `apt install` or `dnf install` etc:

```bash
apt install libmesa opengl or other graphical missing libraries
dnf install libmesa opengl or other graphical missing libraries
```
Issues are further complicated if you are running Windows WSL. Firewall settings, etc.

The problem is getting accelerated graphical capabilities going in these operating systems being a finicky process.

I will convert this to a discussion, as there is nothing we can change in the moose repository that will effect your situation in a positive way.

**Comment by milljm** [comment, marked as answer]
As a troubleshooting step, and a possible solution to reading MOOSE generated output files, lets try to install Paraview. Which using many of the same dependencies as Peacock, but may produce more useful errors!

```bash
conda deactivate
conda activate base
conda create -n paraview paraview
conda activate paraview
paraview
```
It may take over a minute to get some sort of response...
